browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
Cause of this issue with show, hide process of tabs. Refer below URL update hide, show css process. It will work for sure. 

Ref: http://electron.atom.io/docs/api/web-view-tag/#css-styling-notes.